components/AUPage/index.jsx

```diff
@@ -0,0 +1,66 @@
+import Image from 'next/image';
+
+import { Content } from './Content';
+import { CTA } from './CTA';
+import { Hero } from './Hero';
+import { WhatToExpect } from './WhatToExpect';
+
+const data = [
+  {
+    title: 'Cities',
+    value: 5,
+  },
+  {
+    title: 'Sponsors',
+    value: 32,
+  },
+  {
+    title: 'Speakers',
+    value: 68,
+  },
+  {
+    title: 'Attendees',
+    value: '1500+',
+  },
+];
+
+const Numbers = () => (
+  <div className="mt-20 w-full items-end border-y-1.5">
+    <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-4 md:mx-24 lg:mx-48">
+      {data.map((item, index) => {
+        let borderClassName = '';
+        if (index == data.length - 1) borderClassName += 'border-r-1.5';
```

Comment:
always better to compare using three equals (===), doesn't matter here, just for future